docs/guidance/rayservice.md

```diff
@@ -0,0 +1,201 @@
+## Ray Services (alpha)
+
+> Note: This is the alpha version of Ray Services. There will be ongoing improvements for Ray Services in the future releases.
+
+### Prerequisite
+
+* Ray 2.0 is required.
+
+### What is a RayService?
+
+The RayService is a new CR supported by KubeRay in v0.3.0.
+
+A RayService manages 2 things:
+* RayCluster: Manages resources in kubernetes cluster.
+* Rey Serve Deployment Graph: Manages users' serve deployment graph.
+
+### What does the RayService provide?
+
+* Kubernetes-native support for Ray cluster and Ray Serve deployment graphs. Users can use kubernetes config to define ray cluster and ray serve deployment graph. Users can use `kubectl` create ray cluster and ray serve deployment graph.  
```

Comment:
```suggestion
* Kubernetes-native support for Ray cluster and Ray Serve deployment graphs. You can use a kubernetes config to define a ray cluster and its ray serve deployment graphs. Then you can use `kubectl` to create the cluster and its graphs.
```

For documentation, we can use "you" when referring to the user, since the users are the ones reading it. I recommend refactoring the documentation to use "you."